kerl

```diff
@@ -2124,6 +2130,15 @@ index caa1ce568b..6ebb3d3a25 100644
 _END_PATCH
 }
 
+upgrade_kerl() {
+    install_folder=$1
+    wget -q -O "$install_folder/kerl" $KERL_GIT_BASE/kerl
```

Comment:
We use `curl` everywhere else in the script currently, can we please use it here too instead of adding a new dependency on wget.

Reply:
Done in 2d2912a